### PR TITLE
Remove small number hacky fix.

### PR DESCRIPTION
**Pull request by ismailsunni**
### What does it fix ?
* Ticket #3849 
* Funded by : DFAT
* Description : 
   - We no longer need small number as hacky number. We prefer to use > or < operator in the calculation.

### CODE REVIEW

File: safe/definitions/hazard_classifications.py [hunk 1]
```diff
@@ -4,7 +4,7 @@
 See https://github.com/inasafe/inasafe/issues/2920#issuecomment-229874044
 to have a table showing you classes of each kind of hazard.
 """
-from safe.definitions import concepts, small_number
```
Comment (Gustry) [inline safe/definitions/hazard_classifications.py]: It could be nice to add a comment here about we define max and min.

Comment (ismailsunni) [inline safe/definitions/hazard_classifications.py]: Yes, I agree. I will put in the `concepts.py` like @timlinux said in the stand up.

Comment (Gustry) [inline safe/definitions/hazard_classifications.py]: If you can copy/paste also here with comments in this file, it's quicker, for developers.

Comment (ismailsunni) [inline safe/definitions/hazard_classifications.py]: Done.

File: safe/definitions/concepts.py [hunk 2]
```diff
@@ -469,6 +469,33 @@
             }
         ],
     },
+    'thresholds': {
+        'key': 'thresholds',
+        'description': tr(
+            'A range that defined with minimum and maximum value. In InaSAFE '
+            'we exclude the minimum value but include the minimum value. In '
+            'mathematical expression: minimum value < x <= maximum value. It '
+            'is used for doing classification for continuous data.'),
```
Comment (timlinux) [inline safe/definitions/concepts.py]: +            'A range defined with minimum and maximum values. In InaSAFE '
 +            'we exclude the minimum value but include the maximum value. '
 +            'Expressed mathematically: minimum value < x <= maximum value. Ranges '
 +            'are used for performing classification of continuous data.'),

File: safe/definitions/concepts.py [hunk 3]
```diff
@@ -469,6 +469,33 @@
...
+                'link': u''
+            }
+        ],
+    },
+    'value_maps': {
+        'key': 'value_maps',
+        'description': tr(
+            'A mapping of a set of values to another set of unique values. '
+            'Each unique value represents a particular class. It is used for '
+            'doing classification for classified (non-continuous) data'),
```
Comment (timlinux) [inline safe/definitions/concepts.py]: 'A conceptual mapping between one set of unique values and another set of unique values. Each unique value represents a particular class. It is used to express terms or concepts from one classification system in another classification system and only applies to non-continuous data. For example a value map can be used to express local 
names for entities (e.g. street type: "jalan") into generic concepts (e.g. street type: "residential").'

Comment (ismailsunni) [inline safe/definitions/concepts.py]: Done. Thanks @timlinux

**Pull request merged**